lib/quick_travel/booking.rb

```diff
@@ -296,6 +296,11 @@ def secure_access_code
       Encrypt.access_key(@id.to_s)
     end
 
+    def customer_comments
+      comment = comments.select{ |comment| comment['comment_type'] == 'customer' }.first
```

Comment:
Instead of `select{}.first` we can do `detect{}`

Reply:
You still got it @blaknite !